lessons/specifics/ecto.md

```diff
@@ -187,12 +187,24 @@ In addition to `all/2`, Repo provides a number of callbacks including `one/2`, `
 
 ### Count
 
+If we want to count number users that has confirmed account we could use `count/1`:
```

Comment:
It looks like there is a missing preposition: `to count number users` → `to count the number of users`.